src/schema/v2/__tests__/partnerInquirerProfile.test.ts

```diff
@@ -0,0 +1,60 @@
+import gql from "lib/gql"
+import { runAuthenticatedQuery } from "schema/v2/test/utils"
+
+describe("partnerInquirerProfile", () => {
+  const partnerData = {
+    id: "catty-partner",
+    slug: "catty-partner",
+    name: "Catty Partner",
+  }
+
+  const collectorProfile = {
+    name: "Some Collector",
+    location: {
+      city: "Around",
+      country: "The Globe",
+    },
+    profession: "Superhuman",
+    bio: "I got snacks to the roof",
+  }
+
+  const context = {
+    partnerInquirerProfileLoader: () => Promise.resolve(collectorProfile),
+    partnerLoader: () => Promise.resolve(partnerData),
+  }
+
+  it("returns an inquirer's collector profile", async () => {
+    const query = gql`
+      query {
+        partner(id: "catty-partner") {
+          inquiryRequest(inquiryId: "inquiry-id") {
+            collectorProfile {
+              name
+              location {
+                city
+                country
+              }
+              profession
+              bio
```

Comment:
would you mind also adding some of the new fields like `isActiveBidder` `isActiveInquirer` to the test query